_data/osdf_namespace_metadata/spin4d.yaml

```
@@ -14,16 +14,16 @@ description: |
 organization: University of Hawaii-Moana
 dataVisibility: public
 size: 119846767427584
-objectCount:
-bytesXferd: # In bytes so I can parse how I want
-url: http://dtn-itc.ifa.hawaii.edu/spin4d/DR1/
-fieldOfScience: Astronomy and Astrophysics
+objectCount: null
+fieldOfScience: Physical Sciences
 numberOfDatasets: 6
 rank: 2 # Rank of the namespace, 0 will have the lowest priority
 inProgress: false # Whether or not the namespace is in progress
 display: true # Whether or not to display this namespace
 name: SPIn4D Data Release 1
 namespace: 
   - "/uhkoa/SPIN4D-DR1"
-publicObject: # Optional namespace path to a public object
+publicObject: osdf:///uhkoa/SPIN4D-DR1/PIN4D_SSD_50G_V # Optional namespace path to a public object
```

```suggestion
publicObject: /uhkoa/SPIN4D-DR1/SPIN4D_SSD_50G_V # Optional namespace path to a public object
```

Huh that is a listing for me... : http://osdf-director.osg-htc.org/uhkoa/SPIN4D-DR1/SPIN4D_SSD_50G_V